webview.h

```diff
@@ -900,6 +905,9 @@ class win32_edge_engine {
       m_window = CreateWindowW(L"webview", L"", WS_OVERLAPPEDWINDOW,
                                CW_USEDEFAULT, CW_USEDEFAULT, 640, 480, nullptr,
                                nullptr, GetModuleHandle(nullptr), nullptr);
```

Comment:
FYI, one can pass in `hInstance` here instead of calling `GetModuleHandle` again.

Reply:
I've fixed the call to `GetModuleHandle`. I agree about the magic numbers. We can address them in another PR.